src/wp-includes/rest-api/endpoints/class-wp-rest-attachments-controller.php

```diff
@@ -531,7 +531,7 @@ public function edit_media_item( $request ) {
 			);
 		}
 
-		$supported_types = array( 'image/jpeg', 'image/png', 'image/gif', 'image/webp', 'image/avif' );
+		$supported_types = array( 'image/jpeg', 'image/png', 'image/gif', 'image/webp', 'image/avif', 'image/heif' );
```

Comment:
Isn't it `image/heic`? Both seem to be used interchangeably here and there.

Reply:
right, fixing now...